src/NuGet.Core/NuGet.ProjectManagement/Projects/BuildIntegratedNuGetProject.cs

```diff
@@ -204,17 +212,33 @@ public virtual string ProjectName
 
         private async Task<JObject> GetJsonAsync()
         {
-            using (var streamReader = new StreamReader(_jsonConfig.OpenRead()))
+            try
             {
-                return JObject.Parse(await streamReader.ReadToEndAsync());
+                using (var streamReader = new StreamReader(_jsonConfig.OpenRead()))
+                {
+                    return JObject.Parse(await streamReader.ReadToEndAsync());
+                }
+            }
+            catch (Exception ex)
+            {
+                throw new InvalidOperationException(
+                    string.Format(Strings.ErrorLoadingPackagesConfig, _jsonConfig.FullName, ex.Message), ex);
```

Comment:
Is "ErrorLoadingPackagesConfig" the right string resource? We are dealing with project.json and not packages.config, right?